test/conftest.py

```diff
@@ -163,55 +162,6 @@ def pudl_etl_parameters(etl_settings: EtlSettings) -> DatasetsSettings:
     return etl_settings.datasets
 
 
-@pytest.fixture(scope="session", params=["YS"], ids=["ferc1_annual"])
-def pudl_out_ferc1(live_dbs: bool, pudl_engine: sa.Engine, request) -> PudlTabl:
-    """Define parameterized PudlTabl output object fixture for FERC 1 tests."""
-    if not live_dbs:
-        pytest.skip("Validation tests only work with a live PUDL DB.")
-    return PudlTabl(pudl_engine=pudl_engine, freq=request.param)
-
```

Comment:
Got rid of all the PudlTabl fixtures here -- two of them were already not being used.  Now the tests use `pudl.helpers.get_parquet_table()` to read data directly